src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: record_log_event (Python agent API)
+type: apiDoc
+shortDescription: Records a log event for use in logging in context
+tags:
+  - Agents
+  - Python agent
+  - Python agent API
+metaDescription: 'Python API: This call records a log event for use in logging in context.'
+redirects:
+  - /docs/agents/python-agent/python-agent-api/recordlogevent-python-agent-api
```

Comment:
@hmstepanek this redirect looks identical to this new file, which could cause some confusion for Gatsby. I'd like to remove it, unless you see something else going on here.

Reply:
Oops. I copy/paste/modified this from another api doc and I agree I do not think these redirects need to be here. It's ok to remove them!